web_action_conditionable/static/src/js/views.js

```diff
@@ -1,22 +1,21 @@
-/*global openerp, _, $ */
-
-openerp.web_action_conditionable = function (instance) {
-  instance.web.View.include({
-    is_action_enabled: function(action) {
-      var attrs = this.fields_view.arch.attrs;
-      if (action in attrs) {
-	try {
-	  return this._super(action);
-	} catch(error) {
-	  var expr = attrs[action];
-	  var expression = py.parse(py.tokenize(expr));
-	  var cxt = this.dataset.get_context().__eval_context.__contexts[1];
-	  var result = py.evaluate(expression, cxt).toJSON();
-	  return result
+odoo.define("web_action_conditionable.view", function(require) {
+    "use strict";
+    var view = require("web.View");
+    view.include({
+	is_action_enabled : function(action) {
+	    var attrs = this.fields_view.arch.attrs;
+	    if (action in attrs) {
+		try {
+		    return this._super(action);
+		} catch (error) {
+		    var expr = attrs[action];
+		    var expression = py.parse(py.tokenize(expr));
```

Comment:
py Seems to already be in the global namespace as declared on `web/static/lib/py.js/lib/py/js` and whenever is used in the base code in Odoo, require.js is not used. Correct me if I am wrong here since I do not have frontend experience.